### PR TITLE
Report system

### PR DESCRIPTION
**Pull request by kumarprashant24**
Please check the improvement for performance and Schema for report system

### CODE REVIEW

File: models/post.model.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const postSchema = new mongoose.Schema({
         }
       ],
       comment:[{}],
-      report:[{
-        type: Schema.Types.ObjectId, ref: 'user' 
-      }]
+      // report:[{
```
Comment (Mayank461) [inline models/post.model.js:22]: remove the report comments area

**Review by Mayank461 — APPROVED** [review]

File: services/post.service.js [hunk 2]
```diff
@@ -140,19 +144,32 @@ module.exports.comment = async (id, message, user_id, picture_url) => {
...
+      post_uid:data.data._id,
+      post_url: data.data.post_url,
+      posted_by: data.data.posted_by,
+      post_caption:data.data.post_caption,
+      reported_by:report_person
+    }).save();
+
+    return { status: 200 };
+      }
+      else{
```
Comment (mahir-ttn) [inline services/post.service.js:166]: else is not used, can be removed

Comment (kumarprashant24) [reply services/post.service.js:166]: ok i will remove

File: models/post.model.js (hunk 1 above)
Comment (kumarprashant24) [reply models/post.model.js:22]: ok i will remove

**Review by Mayank461 — COMMENTED** [review]
change the report comment area

**Pull request closed without merging**